frontend/src/metabase/public/containers/PublicOrEmbeddedQuestion/index.ts

```diff
@@ -0,0 +1 @@
+export { PublicOrEmbeddedQuestion as PublicOrEmbeddedQuestion } from "./PublicOrEmbeddedQuestion";
```

Comment:
nit: I know this is from using F2 to rename in VSCode. I get this all the time 😆 

```suggestion
export { PublicOrEmbeddedQuestion } from "./PublicOrEmbeddedQuestion";
```